MiniTwit/Program.cs

```diff
@@ -30,6 +31,28 @@
 builder.Services.AddScoped<IPasswordHasher<User>, PasswordHasher<User>>();
 builder.Services.AddMetrics();
 
+
+builder.Services.AddOpenTelemetry().WithMetrics(builder => {
+    builder.AddPrometheusExporter();
+    
+    builder.AddMeter(
+        "Microsoft.AspNetCore.Hosting", 
+        "Microsoft.AspNetCore.Server.Kestrel", 
+        "Microsoft.AspNetCore.Routing", 
+        "Microsoft.AspNetCore.Diagnostics"
+        );
+    
+    builder.AddView("http.server.request.duration",
+       
+        new ExplicitBucketHistogramConfiguration
+        {
+            Boundaries = new double[] { 0, 0.005, 0.01, 0.025, 0.05,
+                    0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 }
```

Comment:
What do these constants mean?

Reply:
Don't know, but they are part of the official documentation: 
https://learn.microsoft.com/en-us/aspnet/core/log-mon/metrics/metrics?view=aspnetcore-8.0

Reply:
Okay, I think they are the buckets for the histogram of the request duraiton